apps/server/src/services/PlaybackService.ts

```diff
@@ -224,4 +224,14 @@ export class PlaybackService {
         : logger.info('PLAYBACK', `Removed ${delayTime} min delay`);
     }
   }
+
+  /**
+   * Updates the duration of the event with the given id
+   * @param {string} eventId
+   * @param {number} newDuration
+   */
+  static updateEventDuration(eventId: string, newDuration: number) {
+    EventLoader.updateEventDuration(eventId, newDuration);
+    logger.info('PLAYBACK', `Updated duration of event with ID ${eventId} to ${newDuration}`);
```

Comment:
The duration value is in milliseconds and would make little sense in most cases. I would suggest either omitting it our running through a format function like `formatDisplay`

Also, we have a LogOrigin enum that would take care of the `'PLAYBACK'` constant string